Rubberduck.Core/UI/CodeExplorer/Commands/AddTemplateCommand.cs

```diff
@@ -59,6 +66,25 @@ private bool SpecialEvaluateCanExecute(object parameter)
             }
         }
 
+        private bool EvaluateCanExecute(ICodeExplorerNode node)
+        {
+            if (!(node is CodeExplorerItemViewModel)
+                || node.Declaration == null)
+            {
+                return false;
+            }
+
+            try
+            {
+                var project = node.Declaration.Project;
+                return AllowableProjectTypes.Contains(project.Type);
```

Comment:
Shouldn't the `Project` be wrapped in a `using`?

Reply:
No, the project on declarations is ultimately managed by the projects repository. Disposing it will break things.